style.css

```diff
@@ -2,6 +2,7 @@
     margin: 0;
     padding: 0;
 }
+
 body{
    background-color: darkorchid;
    font-size: medium;
```

Comment:
- [x] [Optional] You can look at <a href="https://colorhunt.co/">this page</a> to have color palette references to use that improve the presentation of your website. In that way attract and retain the attention of people who see your project. 💥